assets/javascripts/app.coffee

```diff
@@ -67,7 +67,11 @@ prepareForm = (data) ->
   tree = new TreeModel()
   root.starboard.teams = tree.parse(data)
   teamnames = _.map(root.starboard.teams.all(-> true), (team) ->
-    {"name": team.model.id, "slug": team.model.slug}
+    level = team.getPath().length
+    leftpad = ""
+    if level > 1
+      leftpad = Array(level).join("&nbsp;&nbsp;&nbsp;&nbsp; ")
```

Comment:
This is clever but took me a minute to work out what it's doing. The end result is a string containing the joiners between the empty array.

This also means that indentation only start with level 2, since level 1 (`Array(1)`) will not have any joiners.

Either way, maybe something like:

`Array(level).fill('&nbsp;&nbsp;&nbsp;&nbsp; ').join('')`

Would give more of a clue what is going on here.